packages/sitecore-jss/src/editing/graphql-editing-service.test.ts

```diff
@@ -136,6 +143,7 @@ describe('GraphQLEditingService', () => {
       version,
       itemId,
       siteName,
+      layoutKind: LayoutKind.Shared,
```

Comment:
This test case is mixed with the test case "should return empty dictionary and layout". Let's introduce a separate unit test to verify this functionality
"should fetch shared layout editing data"